embedded-cassandra/embedded-cassandra-war/pom.xml

```diff
@@ -26,41 +26,50 @@
     <version>0.6.1.Final-SNAPSHOT</version>
   </parent>
 
-  <artifactId>hawkular-commons-embedded-cassandra-ear</artifactId>
-  <packaging>ear</packaging>
```

Comment:
> why this was previously packaged as EAR

I have no idea myself ( @stefannegrea might know ) and I see no necessity for EAR packaging. As I said the EAR plugin is buggy, and I therefore want to avoid it.

> why EJB's had to be removed

Perhaps they do not need to be removed. Anyway, they can be removed and CDI seems to be more common than EJBs among the Hawkular components.